### DISCUSSION TITLE
how to get the .csv file to use dataset_from_csv.py ?

### DISCUSSION THREAD

**Discussion by zhuliyi0** [bghira/SimpleTuner#129] [Q&A]
Did a quick search, didn't find a clue. Can you point to a direction? @bghira Thanks!

I am planning to use some laion data that is completely unrelated to the training subject type as regularization, and see if it helps or not.

**Comment by bghira** [comment]
dataset_from_csv is a helper script that downloads images locally. it has some hardcoded paths in it that you'll need to update before it is run.

**Comment by bghira** [comment]
if you're interested, the csv_to_s3.py file can be used as a basis to bring the dataset_from_csv.py up to better standards, with args parsing and more. ideally they would use common helper methods instead of being standalone scripts, as the scope/goal of this project has shifted from having purely standalone scripts that do not require the rest of the project.
